### DISCUSSION TITLE
DWFC wannier.json examples and data.wfc

### DISCUSSION THREAD

**Discussion by doylecc416** [deepmodeling/deepmd-kit#154] [Q&A]
In the /examples dir for water, there is an input .json file ( wannier.json ). Is this file corresponding to the _"Deep neural network for Wannier function centers"_ paper (https://arxiv.org/pdf/1906.11434.pdf) / DWFC methods? If so, what type of data, and in what format should the /data.wfc/wfc.npy be in? The explanation for box.npy, coord.npy, etc. is thorough, but wfc.npy is not explained. If not, what does this input file do? Are there any plans to provide an example of the DWFC scheme?

**Comment by njzjz** [comment]
@jameswind

**Comment by jameswind** [comment]
Thank you for the reminder. I am preparing and testing the example and will
add corresponding data by this weekend.

**Comment by doylecc416** [comment]
Thank you for the reply. Is the example that you are preparing for the combination of DPMD with DWFC (for the spectral calculation in the paper)? This would make me think that the current wannier.json is just for calculating the pair correlations and dipoles, not using the correct time evolution snapshots that DPMD would provide.

**Comment by jameswind** [comment]
The current wannier.json is just for training. Calculations of the DWFC
positions and other properties are left to post-processing procedures.

**Comment by doylecc416** [comment]
Where will the example be posted?
